js/Ice/fileSystem/client.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) ZeroC, Inc.
+
+import { Ice } from "@zeroc/ice";
+import { Filesystem } from "./Filesystem.js";
+
+// Create an Ice communicator to initialize the Ice runtime. The communicator is disposed before the program exits.
+await using communicator = Ice.initialize(process.argv);
+
+// Create a proxy for the root directory.
+const rootDir = new Filesystem.DirectoryPrx(communicator, "RootDir:tcp -h localhost -p 4061");
+
+// Recursively list the contents of the root directory.
+console.log("Contents of root directory:");
+await listRecursive(rootDir);
+
+/**
```

Comment:
Can we use `///` for doc-comments in TS?

Reply:
No. TypeScript uses JSDoc style doc comments.

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html